src/redux/books/booksSlice.js

```diff
@@ -1,41 +1,103 @@
+/* eslint-disable consistent-return */
+/* eslint-disable no-unused-vars */
 /* eslint-disable no-param-reassign */
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+import restructureApiResponse from '../../utils/restructureApiResponse';
+
+const baseAPI = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/ATwGr9HsrW4O0MQXnRyQ/books';
+
+export const getBooks = createAsyncThunk(
+  'books/getBooks',
```

Comment:
This may be trivial but in real world experience, you need to stick to the naming conventions agreed upon by the company and your colleagues, failure to do so may cause a lot of confusion. For instance, the project requirements expects you to create a special action creator using [createAsyncThunk](https://redux-toolkit.js.org/api/createAsyncThunk) called `fetchBooks`. I can see that you have named it differently. Kindly consider sticking to the naming conventions in the project requirments as shown below: :point_down: 

![Screenshot from 2023-06-15 06-38-27](https://github.com/ybadr99/bookstore/assets/65233860/224bd18e-b0f8-4d3c-9637-0db0bd4274c1)